### ISSUE TITLE
Setup information

### ISSUE THREAD

**Issue by sranger** [mattgwagner/Dreamhost-Dynamic-DNS-Updater#4]
Some information on how to set this up would be helpful. 

It looks like some information is needed in the Service/App.config (api key, dh user, hostname, logfile) but it's not defined anywhere. Also, is there any setup I need to do in my DH account? It seems I need the API key but anything else? I think I have it all set up on my pc but not sure if I need to define the subdomain before it will work correctly; if so, info on how to do that would be helpful. Also, the items necessary for the API key to have enabled would be useful, too.

Other than that, thanks for the very helpful project; hopefully my initial attempt at getting this working works okay and I don't need the info above!

**Comment by sranger** [comment]
Edit: looks like you need to add an A entry into your DNS for your domain (which the service will update). Adding this into the readme would help.

**Comment by mattgwagner** [comment]
I agree. If you can, click the edit button on the readme and submit some
updates for what helped you get rolling with it. I've needed to tweak the
documentation for some time....
